packages/neutrino-middleware-start-server/index.js

```diff
@@ -1,3 +1,13 @@
 const StartServerPlugin = require('start-server-webpack-plugin');
 
-module.exports = ({ config }, options) => config.plugin('start-server').use(StartServerPlugin, [options.name]);
+module.exports = (neutrino, options) => {
```

Comment:
Yes that worked, after I removed the getter/setters for the path options in neutrino api.js